comp/core/workloadmeta/types.go

```diff
@@ -340,6 +340,8 @@ type ContainerPort struct {
 	Name     string
 	Port     int
 	Protocol string
+	HostPort uint16
+	HostIP   string
```

Comment:
why do we need hostIP here? 
can we have different hostIP in containers on the same host? 
Should we put instead the information in a host entity? or on the task level?

Reply:
I deleted this field